### PR TITLE
Add Kurdish Layout

### PR DESCRIPTION
**Pull request by GoRaN909**
Add Kurdish Keyboard Layout Qwerty Based Without Shift.

### CODE REVIEW

**Review by Julow — APPROVED** [review]
Thanks ! I'll leave this open until the next release to allow you to make additional changes. 

I left some comments. Sorry for the slow reply.

**Comment by Julow** [comment]
Are you happy with the bad alignment I mentioned above ? I plan to merge this then release soon.

File: res/xml/kurdish.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <key key0="ز" key1="ظ"/>
+    <key key0="خ"/>
+    <key key0="ج"/>
+    <key key0="چ"/>
+    <key key0="ح" key1="&#1567;" key3="!"/>
+    <key key0="ع" key1="ﷲ" key2="ﷻ"/>
+    <key key0="ب" key2="﷽"/>
+    <key key0="ن" key2="&#1548;" key3="&#1563;"/>
+    <key key0="م" key2="ﷴ" key3="ﷺ"/>
+    <key key0="backspace" key2="delete" key3="f11_placeholder" key4="f12_placeholder" width="1.333"/>
```
[the remark this answers is not in the document]
Comment (GoRaN909) [reply res/xml/kurdish.xml:39]: Hi sorry for late replay and thanks for you suggest I just modified cantered the first and second row also removes unused unicode charterers.  

And about layout name is just okay no need to change I think now you can merge into master branch. 

Thanks.

**Comment by GoRaN909** [comment]
> Are you happy with the bad alignment I mentioned above ? I plan to merge this then release soon.

I just fixed the width of first and second row now is can be merge. Thanks

File: res/xml/kurdish.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key shift="0.1665"/>
+    <key key0="ق" key1="ٯ" key2="١" key3="esc"/>
```
Comment (Julow) [inline res/xml/kurdish.xml:5]: This was creating a key with no symbol on it. You meant this:
```suggestion
    <key shift="0.1665" key0="ق" key1="ٯ" key2="١" key3="esc"/>
```

File: res/xml/kurdish.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <key key0="ت" key1="ط" key2="٥" key3="٪"/>
+    <key key0="ی" key1="ي" key2="٦" key3="^"/>
+    <key key0="ێ" key1="ؽ" key2="٧" key3="&amp;"/>
+    <key key0="ئ" key1="ء" key2="٨" key3="*"/>
+    <key key0="ۆ" key1="ؤ" key2="٩" key3=")" key4="("/>
+    <key key0="پ" key1="ث" key2="٠"/>
+  </row>
+  <row>
+    <key shift="0.1665"/>
+    <key key0="ا" key1="أ" key2="tab"/>
```
Comment (Julow) [inline res/xml/kurdish.xml:18]: Same as the first row.
```suggestion
    <key shift="0.1665" key0="ا" key1="أ" key2="tab"/>
```

Comment (GoRaN909) [reply res/xml/kurdish.xml:17]: My language is RTL we don't need/use the `Shift` button to uppers & lower case!

Comment (Julow) [reply res/xml/kurdish.xml:17]: This is not the `Shift` button but an option to add some space on the left of the keyboard.

Comment (GoRaN909) [reply res/xml/kurdish.xml:17]: Oh that's great so we are now ready to merge it :)  

I tried to build an debug apk file with Android studio but i doesn't sync with gradle it looks like not the `Java` language code!

Comment (Julow) [reply res/xml/kurdish.xml:17]: Alright, I'll merge! 

I have not tried using Android Studio but if you find a way to make it work, I'd be happy to merge it. The project currently uses a Makefile.
I don't want to use gradle for reproducibility purposes.

**Comment by Julow** [comment]
In your commit you mention:

> I just want to use same width for all rows just like the third one with full fit no space on the sides bro the third row is correct one like all other keyboards.

This is not what happen due to the `width` option of the backspace key I warned you about. Here's how it looks now: (some keys are added automatically just for me, ignore them)

shot

Here's how it would look with a smaller `backspace` key:

shot

**Comment by GoRaN909** [comment]
> In your commit you mention:
> 
> > I just want to use same width for all rows just like the third one with full fit no space on the sides bro the third row is correct one like all other keyboards.
> 
> This is not what happen due to the `width` option of the backspace key I warned you about. Here's how it looks now: (some keys are added automatically just for me, ignore them)
> 
> shot
> 
> Here's how it would look with a smaller `backspace` key:
> 
> shot

You just do a great by use a smaller backspace now all buttons are equal and very good looking thanks for the changes it's time to merge i really want to use this great keyboard in my native language.

**Comment by Julow** [comment]
Let's merge!

**Comment by GoRaN909** [comment]
> Let's merge!

Wow it's perfect thanks for merging bro 🥰

Screenshot_20230605_012026_Unexpected Keyboard (debug)

**Pull request merged**